pkg/kubeadm/master/manifests.go

```diff
@@ -216,7 +216,8 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 			"--cluster-signing-cert-file=" + pki_dir + "/ca.pem",
 			"--cluster-signing-key-file=" + pki_dir + "/ca-key.pem",
 			"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
-			"--cluster-cidr=" + s.InitFlags.Services.CIDR.String(),
+			"--allocate-node-cidrs=true",
+			"--cluster-cidr=" + s.InitFlags.PodNetwork.CIDR.String(),
```

Comment:
Also, I have to say that not separating these two flags seems natural, but it'd be good to add a comment on the reasoning why we grouped these two (stating what are the use-cases when you want to set it, but don't want to allocate it).